docs/usage/getting-started/running.md

```diff
@@ -153,100 +153,16 @@ Regardless of platform, you need to select a user account for `renovate` to assu
-
-#### Bitbucket Cloud
-
-First, [create an AppPassword](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/) for the bot account.
-Give the bot App password the following permission scopes:
-
-- [`account`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#account) (Account: Read)
-- [`team`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#team) (Workspace membership: Read)
-- [`issue:write`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#issue-write) (Issues: Write)
-- [`pullrequest:write`](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#pullrequest-write) (Pull requests: Write)
-
-The bot also needs to be able to validate the workspace membership status of pull-request reviewers, for that, [create a new user group](https://support.atlassian.com/bitbucket-cloud/docs/organize-workspace-members-into-groups/) in the workspace with the **Create repositories** permission and add the bot user to it.
-
-Configure it as `password` in your `config.js` file, or in environment variable `RENOVATE_PASSWORD`, or via CLI `--password=`.
-Also be sure to configure the `username` for your bot account too.
-Don't forget to configure `platform=bitbucket` somewhere in config.
-
-#### Bitbucket Server
-
-Create a [Personal Access Token](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html) for your bot account.
-Configure it as `password` in your `config.js` file, or in environment variable `RENOVATE_PASSWORD`, or via CLI `--password=`.
-Also configure the `username` for your bot account too, if you decided not to name it `@renovate-bot`.
-Don't forget to configure `platform=bitbucket-server` somewhere in config.
-
-If you use MySQL or MariaDB you must set `unicodeEmoji` to `false` in the bot config (`RENOVATE_CONFIG_FILE`) to prevent issues with emojis.
-
-#### Azure DevOps
-
-First, [create a Personal Access Token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page) for the bot account.
-Configure it either as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
-Don't forget to configure `platform=azure` somewhere in config.
-
-#### Gitea
-
-First, [create an access token](https://docs.gitea.io/en-us/api-usage/#authentication-via-the-api) for your bot account.
-Configure it as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
-Don't forget to configure `platform=gitea` somewhere in config.
+- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
+- [Bitbucket Cloud](https://docs.renovatebot.com/modules/platform/bitbucket/)
+- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
+- [Gitea](https://docs.renovatebot.com/modules/platform/gitea/)
+- [github.com or GHES](https://docs.renovatebot.com/modules/platform/github/)
```

Comment:
```suggestion
- [github.com and GitHub Enterprise](https://docs.renovatebot.com/modules/platform/github/)
```

Maybe we should call it something like this?

The page for `modules/platform/github/index` has the title `GitHub and GitHub Enterprise`:

https://docs.renovatebot.com/modules/platform/github/

Reply:
Most people call GHES "GitHub Enterprise" for short, but technically you can be GHE on either cloud or server (self-hosted). Maybe we fix both

Reply:
I don't understand what you want me to do to "fix both". 🙈 Can you explain what you want me to change and where?

GitHub calls their offering: "Enterprise Cloud" and "Enterprise Server".

Reply:
I mean to use the term "GitHub Enterprise Server" in all places we mention it

Reply:
Just to add on that, this is mainly based on a rebranding from GitHub. A couple of years ago there was `GitHub` for github.com and `GitHub Enterprise` for their on-premise offering. They then probably wanted to address the Enterprise Cloud market and started to name their Cloud product aimed at Enterprises `GitHub Enterprise Cloud` (which is just a fancy contract model for github.com) and their on-premise offering got renamed from just `GitHub Enterprise` to `GitHub Enterprise Server` causing quite a bunch of confusion along the way. 
So now you have:
- `GitHub` for the normal github.com experience
- `GitHub Enterprise Cloud` for a contract model on github.com aimed at Enterprises (from a integration point the same as `GitHub`)
- `GitHub Enterprise Server` for their on-premise offering with custom domain and fully isolated setup.

Reply:
@Shegox Thanks for the explanation of the marketing history for "GitHub on premises". ❤️